cyclonedx-lib/build.xml

```diff
@@ -69,6 +69,31 @@
                 </copy>
         </target>
 
+        <target name="buildSignSBOM">
+                <mkdir dir="build/sign_classes"/>
```

Comment:
"buildSignSBOM" needs to depend on the openkeystore and cyclonedx targets, eg try this:
```
	<target name="build-sign-sbom" depends="dep-checks, build-openkeystore, download-cyclonedx, download-jackson-core, download-jackson-dataformat-xml, download-jackson-databind, download-jackson-annotations, download-json-schema, download-commons-codec, download-commons-io, download-github-package-url, compile-sign-sbom, jar-sign-sbom">
                <echo message="Building cyclonedx-lib TemurinSignSBOM"/>
        </target>

        <target name="compile-sign-sbom">
                <mkdir dir="build/sign_classes"/>
                <javac srcdir="sign_src" destdir="build/sign_classes" classpath="build/jar/cyclonedx-core-java.jar:build/openkeystore/library/dist/webpki.org-libext-1.00.jar" includeantruntime="false"/>
                <javac debug="true" debuglevel="lines,vars,source" srcdir="sign_src" destdir="build/sign_classes" classpath="build/jar/cyclonedx-core-java.jar:build/openkeystore/library/dist/webpki.org-libext-1.00.jar" includeantruntime="false"/>
        </target>

        <target name="jar-sign-sbom">
                <mkdir dir="build/sign_jar"/>
                <jar destfile="build/sign_jar/temurin-sign-sbom.jar" basedir="build/sign_classes">
                        <manifest>
                                <attribute name="Main-Class" value="temurin.sbom.TemurinSignSBOM"/>
                        </manifest>
                </jar>
        </target>
```